src/components/cl/hier/alltoallv/alltoallv.c

```diff
@@ -250,10 +272,16 @@ UCC_CL_HIER_PROFILE_FUNC(ucc_status_t, ucc_cl_hier_alltoallv_init,
     *task = &schedule->super;
     return UCC_OK;
 
+err_init_3:
+    if (!full_only) {
```

Comment:
minor: if im not wronf this can only be accessed if full_only != 0

Reply:
if full only == 0 and error occurs in line 259 or 262 needs to go to error_2 but if full_only != 0 same error needs to go to error 3. If I dont always send to error 3 with if statement then I would need to define separate goto (error 2 or 3)